### DISCUSSION TITLE
Cannot read properties of undefined (reading 'split')

### DISCUSSION THREAD

**Discussion by rlourette** [community/community#103190] [Copilot Conversations]
### Select Topic Area

Bug

### Body

No matter what the prompt, this message is always returned. I relaunched Visual code. Still not working .
`Cannot read properties of undefined (reading 'split')`

**Comment by rlourette** [comment]
I went to uninstall and reinstall copilot plug in and it was reporting a reload was required (sorry I don' recall exactly), but after the reload. No more "Cannot read properties of undefined (reading 'split')".

**Reply by Samabena** [reply]
Great , please I'm not sure to understand your solution. I have similar issue , please can you help me out ? 

When I run one of the following test example , I'm still getting the below error , any help ? 
`(index):29 Uncaught TypeError: Cannot read properties of undefined (reading 'isCompressed') `
